integration/tests/tracer/test_tracer_historical_methods.py

```diff
@@ -0,0 +1,293 @@
+            self.sender_account.address)
+        recipient_balance = self.get_balance_from_wei(
+            self.recipient_account.address)
+
+        if request_type == "blockNumber":
+            request_value = hex(reciept_1[request_type])
+        else:
+            request_value = reciept_1[request_type].hex()
+
+        wait_condition(lambda: int(self.tracer_api.tracer_send_rpc(method="eth_getBalance",
+                                                                   req_type=request_type,
+                                                                   params=[self.sender_account.address,
+                                                                           {request_type: request_value}])["result"], 0) / 1e18 == sender_balance,
+                       timeout_sec=120)
+
+        wait_condition(lambda: int(self.tracer_api.tracer_send_rpc(method="eth_getBalance",
+                                                                   req_type=request_type,
+                                                                   params=[self.recipient_account.address,
+                                                                           {request_type: request_value}])["result"], 0) / 1e18 == recipient_balance,
+                       timeout_sec=150)
+
+        reciept_2 = self.send_neon(
+            self.sender_account, self.recipient_account, transfer_amount)
+
+        sender_balance_after = self.get_balance_from_wei(
+            self.sender_account.address)
+        recipient_balance_after = self.get_balance_from_wei(
+            self.recipient_account.address)
+
+        if request_type == "blockNumber":
+            request_value = hex(reciept_2[request_type])
+        else:
+            request_value = reciept_2[request_type].hex()
+
+        wait_condition(lambda: int(self.tracer_api.tracer_send_rpc(method="eth_getBalance",
+                                                                   req_type=request_type,
+                                                                   params=[self.sender_account.address,
+                                                                           {request_type: request_value}])["result"], 0) / 1e18 == sender_balance_after,
+                       timeout_sec=120)
+
+        wait_condition(lambda: int(self.tracer_api.tracer_send_rpc(method="eth_getBalance",
+                                                                   req_type=request_type,
+                                                                   params=[self.recipient_account.address,
+                                                                           {request_type: request_value}])["result"], 0) / 1e18 == recipient_balance_after,
+                       timeout_sec=150)
+
+    def test_eth_get_code(self):
+        request_type = "blockNumber"
+
+        nonce = self.web3_client.eth.get_transaction_count(
+            self.sender_account.address
+        )
+        transaction = {
```

Comment:
you can use BaseTests.make_tx_object here too

Reply:
done